src-self-hosted/codegen/c.zig

```diff
@@ -142,6 +160,8 @@ fn genFn(file: *C, decl: *Decl) !void {
             if (switch (inst.tag) {
                 .assembly => try genAsm(&ctx, inst.castTag(.assembly).?),
                 .call => try genCall(&ctx, inst.castTag(.call).?),
+                .add => try genBinOp(&ctx, inst.cast(Inst.BinOp).?, "+"),
+                .sub => try genBinOp(&ctx, inst.cast(Inst.BinOp).?, "-"),
```

Comment:
This is good for now but note that these instructions are soon going to change to:
 * `add_wrap`
 * `add_nowrap`
 * `sub_wrap`
 * `sub_nowrap`

`+` and `-` in C will only work for signed nowrap and unsigned wrap. For the others we will want a macro definition that communicates the UB to the C compiler in the case of unsigned nowrap, and performs wrapping in the case of signed wrap.

Reply:
That can come once the instructions are changed :)